extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -256,6 +278,10 @@ private Properties getSchedulerConfigurationProperties(QuartzSupport quartzSuppo
                 props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".isClustered", "true");
                 props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".clusterCheckinInterval", "20000"); // 20 seconds
             }
+
+            if (buildTimeConfig.storeType.clazz.equals(JobStoreCMT.class.getName())) {
```

Comment:
I'd rather use the enum constants, i.e. `buildTimeConfig.storeType == DB || buildTimeConfig.storeType == JDBC_CMT`.